buildtest/cli/__init__.py

```diff
@@ -876,7 +876,7 @@ def report_menu(subparsers):
     subparsers.add_parser("clear", help="Remove all report file")
     subparsers.add_parser("list", help="List all report files")
     parser_report_summary = subparsers.add_parser(
-        "summary", help="Summarize test report"
+        "summary", aliases=["sm"], help="Summarize test report"
```

Comment:
I appreciate your suggestion for adding an alias `sm` for `buildtest rt summary` but i would *strongly* recommend you dont mix up two features into same PR. This PR was intended to remove `--pager` but its also added another feature. Please create a separate PR for this support. Might as well create an issue first if you have any ideas before implementing them.